main/src/ui/chat_input/chat_input_controller.vala

```diff
@@ -48,12 +48,32 @@ public class ChatInputController : Object {
         chat_text_view_controller.initialize_for_conversation(conversation);
         
         Xmpp.Jid? own_jid = stream_interactor.get_module(MucManager.IDENTITY).get_own_jid(conversation.counterpart, conversation.account);
+        
         if (conversation.type_ == conversation.Type.GROUPCHAT){
             if (stream_interactor.get_module(MucManager.IDENTITY).is_moderated_room(conversation.account, conversation.counterpart) && 
             stream_interactor.get_module(MucManager.IDENTITY).get_role(own_jid, conversation.account)==Xmpp.Xep.Muc.Role.VISITOR) {
-                set_input_field_status(new Plugins.InputFieldStatus("This is a moderated group, you don't have voice", Plugins.InputFieldStatus.MessageType.ERROR, Plugins.InputFieldStatus.InputState.NO_SEND));
+                set_input_field_status(new Plugins.InputFieldStatus("This conference does not allow you to send messages. <a href=\"nonexistant\"> Request permission </a>", Plugins.InputFieldStatus.MessageType.ERROR, Plugins.InputFieldStatus.InputState.NO_SEND, true));
```

Comment:
i18n: Make strings translatable by enclosing them in `_()`. In this specific case, please make two translatable strings such that the link markup is not in one of them, e.g.
```vala
_("This conference does not allow you to send messages. %s").printf("<a href=\"x-dino:open-conversation-details\">" + _("Request permission") + "</a>")
```